allensdk/brain_observatory/nwb/nwb_api.py

```diff
@@ -43,7 +43,7 @@ def from_path(cls, path, **kwargs):
         try:
             with open(path, 'r'):
                 pass
-        except Exception as err:
+        except Exception:
```

Comment:
??

Reply:
err was unused here.

Reply:
looking at this again, I have no idea why this try except is here. removing ...